src/common/components/vanilla/button/button-positive.js

```diff
@@ -0,0 +1,15 @@
+import React, { PropTypes } from 'react';
+import styles from './button-positive.css';
+
+export default function ButtonPositive(props) {
+  return (
+    <button className={ styles['button-positive'] } onClick={ props.onClick }>
+      { props.children }
+    </button>
+  );
+}
+
+ButtonPositive.propTypes = {
+  children: PropTypes.node,
+  onClick: PropTypes.func
```

Comment:
It would be better to add vanilla styles to current `Button` implementation I guess.
We already cover more cases and props there (disabled, type for forms, href for anchor).